snomed/com.b2international.snowowl.snomed.importer/src/com/b2international/snowowl/snomed/importer/net4j/SnomedImportRequest.java

```diff
@@ -69,16 +70,23 @@ protected void requesting(final ExtendedDataOutputStream out, final OMMonitor mo
 			}
 			
 			out.writeUTF(importConfiguration.getCodeSystemShortName());
-			
+			final Collection<File> descriptionsFiles = importConfiguration.getDescriptionsFiles();
+			final Collection<File> languageRefSetFiles = importConfiguration.getLanguageRefSetFiles();
 			monitor.worked(); // 1
 			
+			for (File descfile : descriptionsFiles) {
+				writeComponent(out, descfile, monitor);
```

Comment:
This is not going to work like this, see how refset URLs or excluded refset ids are sent over the wire.